js/src/data/selectors.js

```diff
@@ -70,6 +70,20 @@ export const getSettings = ( state ) => {
  * @property {number} [per_page] The number of items per page.
  */
 
+/**
+ * Error object returned from the API.
```

Comment:
@ankitrox No need to define `ApiError` here again. You can import it from `actions.js`. For e.g
```
* @typedef {import('./actions').ApiError} ApiError
```